pandas/io/sql.py

```diff
@@ -750,7 +750,8 @@ def _get_column_names_and_types(self, dtype_mapper):
             for i, idx_label in enumerate(self.index):
                 idx_type = dtype_mapper(
                     self.frame.index.get_level_values(i))
-                column_names_and_types.append((idx_label, idx_type, True))
+                column_names_and_types.append(((text_type(idx_label)),
```

Comment:
Three lines further down the same kind of type_text conversion was done with a set of extra brackets. I just copied the style for consistency.